### PR TITLE
Create design-handoff

### PR DESCRIPTION
**Pull request by akemalakcay**

### CODE REVIEW

**Comment by alpcanaydin** [comment]
@akemalakcay I added the ".md" extension to file :)

File: checklists/design-handoff.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+We use layer styles for every layers on our design files. Every text, every layer needs to have proper layer/text style.
+
+- [ ] Update style guide  
+Style guide is one of the main things that keeps the UI consistent. You need to update the style guide after creating a new component because every componenent needs to be in the style guide so that both developers and other designers can easily work on the project.
+
+- [ ] Mark exportables ready  
+We need to mark our image assets (icons, logos, images) exportables so that developers can download them within the Abstract.
+
+- [ ] Create a collection on Abstract  
+We use collections feature of Abstract. Every project needs to have a style guide collection along with the collections of every features.
```
Comment (alpcanaydin) [inline checklists/design-handoff.md]:
```suggestion
- [ ] Create a collection on Abstract  
We use collections feature of Abstract. Every project needs to have a style guide collection along with the collections of every features. 
```

**Comment by alpcanaydin** [comment]
@akemalakcay You may think to add "handoff" tag  :)

**Comment by akemalakcay** [comment]
> @akemalakcay You may think to add "handoff" tag :)

Nice thinking, I'll add it @alpcanaydin thanks!

**Pull request merged**